site/en/blog/private-network-access-update/index.md

```diff
@@ -1,61 +1,89 @@
 ---
 layout: 'layouts/blog-post.njk'
-title: Private Network Access (CORS-RFC1918) updates
+title: "Private Network Access update: Introducing Deprecation Trial"
 authors:
-  - agektmr
   - titouan
-description: Chrome has some updates to share about the standardization of the Private Network Access - formerly known as CORS-RFC1918 - plans to restrict private network resource accesses, and how your websites can align with those changes.
+description: Chrome is deprecating access to private network endpoints from non-secure websites in Chrome 93 as part of the Private Network Access specification. Read on for recommended actions.
 date: 2021-03-11
```

Comment:
```suggestion
date: 2021-07-29
```

Reply:
I think it's better to bump this date, so the post is resurfaced on /blog

Reply:
This sounds good to me.